tests/pr_test.go

```diff
@@ -2,43 +2,88 @@
 package test
 
 import (
+	"fmt"
+	"log"
+	"os"
+	"strings"
 	"testing"
 
+	"github.com/gruntwork-io/terratest/modules/random"
 	"github.com/stretchr/testify/assert"
 	"github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper/testhelper"
+	"gopkg.in/yaml.v3"
 )
 
-// Use existing resource group
-const resourceGroup = "geretain-test-resources"
-const defaultExampleTerraformDir = "examples/default"
+// Resource groups are maintained https://github.ibm.com/GoldenEye/ge-dev-account-management
+// Allow the tests to create a unique resource group for every test to ensure tests do not clash. This is due to the fact that the auth policy created by this module has to be scoped to the resource group (see https://github.ibm.com/GoldenEye/client-to-site-vpn-module/pull/303#issuecomment-54128819) and hence would clash if tests used same resource group.
+// const resourceGroup = "geretain-test-client-to-site-vpn"
+const highAvailabilityModeExampleTerraformDir = "examples/highavailability_mode"
+const certificateTemplateName = "geretain-cert-template"
 
-func setupOptions(t *testing.T, prefix string, dir string) *testhelper.TestOptions {
+var vpnClientAccessGroupUsers = []string{"GoldenEye.Development@ibm.com"}
```

Comment:
Can we create any other users? We probably don't want to push GoldenEye email ID to the public module

Reply:
I am wondering if we can change that to something generic rather than the actual email ID. I think any email ID would work for the test right?

Reply:
or perhaps it has to be a valid user in the account?